web/api/report_server.thrift

```diff
@@ -396,6 +396,17 @@ struct CheckerCount {
 }
 typedef list<CheckerCount> CheckerCounts
 
+struct CheckerInfo {
+  1: string       checkerName,  // Analyzer name of the checker.
+  2: string       analyzerName, // Analyzer name of the checker.
+  3: Severity     severity,     // Severity level of the checker.
+  4: list<i64> enabled,      // Runs' names in which the checker enabled.
+  5: list<i64> disabled,     // Runs' names in which the checker disabled.
+  6: i64          closed        // Number of closed reports.
+  7: i64          outstanding   // Number of outstanding reports.
+}
+typedef map<i64, CheckerInfo> CheckerInfos
```

Comment:
What is the key int here?

Reply:
The key for CheckerInfo is a checker id.